code/game/gamemodes/cult/runes.dm

```diff
@@ -460,7 +487,7 @@ var/list/teleport_runes = list()
 
 //Ritual of Dimensional Rending: Calls forth the avatar of Nar-Sie upon the station.
 /obj/effect/rune/narsie
-	cultist_name = "Tear Reality (God)"
+	cultist_name = "Summon Your God"
```

Comment:
i can the only reason i did it is..someone summoned the wrong summoning rune on a cult round so...